.idea/Easier68k.iml

```diff
@@ -2,7 +2,7 @@
 <module type="PYTHON_MODULE" version="4">
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$" />
-    <orderEntry type="jdk" jdkName="Python 3.5.2 (/usr/bin/python3.5)" jdkType="Python SDK" />
+    <orderEntry type="jdk" jdkName="Python 3.6" jdkType="Python SDK" />
```

Comment:
maybe we should add this file to the gitignore, or instead setup all of the environment settings that need to once to avoid this coming back

Reply:
I agree. The other way to do it is using the ignore file in `.git/info/exclude`

Reply:
That would change the setting for only your local repo, it's great for IDE files like that